src/mono/wasm/Wasm.Build.Tests/TestAppScenarios/AppSettingsTests.cs

```diff
@@ -13,7 +14,7 @@
 
 namespace Wasm.Build.Tests.TestAppScenarios;
```

Comment:
nit: We can drop the folder `TestAppScenarios`, these tests are no longer different, right?